scipy/stats/tests/test_stats.py

```diff
@@ -2875,19 +2883,19 @@ def test_ttest_rel():
     y = (stats.norm.rvs(loc=5, scale=10, size=501) +
          stats.norm.rvs(scale=0.2, size=501))
     y[500] = np.nan
-    with warnings.catch_warnings():
-        warnings.filterwarnings('ignore', category=RuntimeWarning)
+
+    with np.errstate(invalid="ignore"):
```

Comment:
No, I don't find it very strange, I just expected it more on windows or so, not so much on travis (to see some I don't see locally).